pixl_core/pyproject.toml

```diff
@@ -23,6 +23,7 @@ dependencies = [
     "sqlalchemy==2.0.24",
     "psycopg2-binary==2.9.9",
     "pandas==1.5.1",
+    "pyarrow==14.0.1",
```

Comment:
I'm assuming we need this for pandas?

Reply:
Yeah, writing parquet files specifically. The dummy parquet writing we used to do doesn't create the needed directories so I thought I'd just call the real function instead.